graphp/core/GPLibrary.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+class GPLibrary extends GPController {
+
+  public function isAllowed(string $method): bool {
+    if (GP::isCLI()) {
+      return true;
+    }
+    return false;
```

Comment:
What is the logic here? Is it the case that all the methods of a GPLibrary should be runnable from the CLI?

Reply:
Yep, by default Library allows CLI (which is needed for stuff like `async`), but not accessible to the user trough the browser. This makes it easier to test libraries, write one offs, etc without worrying about exposing the library to an outside user.

Reply:
👍